### PR TITLE
Issue4 client v2

### PR DESCRIPTION
**Pull request by shvets-semandex**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 1.1.0
```
Comment (djmakwana) [inline CHANGELOG.md]: Let's make this also 2.0.0

**Review by djmakwana — COMMENTED** [review]
Are the tests passing? Most of the changes are changing the test framework and classes. Is that introduced by us?

File: build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
+  kotlinOptions.jvmTarget = JavaVersion.VERSION_17.toString()
+}
+
+dependencies {
+  implementation("org.jetbrains.kotlin:kotlin-reflect:$kotlin_version")
+
+  implementation("org.opensearch:opensearch:$openSearchVersion")
+  implementation("org.opensearch.client:opensearch-rest-high-level-client:$openSearchVersion")
```
Comment (chirag4semandex) [inline build.gradle.kts:42]: Do we even use this? If not, can you just comment this and see if the test works?

**Review by chirag4semandex — COMMENTED** [review]
I dont see any impact of the client change, can you check if it really uses the high level client?

File: build.gradle.kts (hunk 2 above)
Comment (shvets-semandex) [reply build.gradle.kts:42]: yes, it uses some of classes, like HasChildQueryBuilder or HasParentQueryBuilder

**Review by chirag4semandex — APPROVED** [review]

**Review by djmakwana — APPROVED** [review]

**Pull request merged**